cypress.json

```diff
@@ -1 +1,10 @@
-{}
+{
+    "viewportWidth": 1200,
+    "viewportHeight": 1200,
+    "responseTimeout": 80000,
+    "defaultCommandTimeout": 30000,
+    "requestTimeout": 20000,
+    "screenshotOnRunFailure": false,
+    "video": false,
+    "baseUrl": "http://localhost:8000/data-research/consumer-complaints/search/"
```

Comment:
this is applicable if you run it through CFGOV and port 8000.  sometimes, I run ccdb-ui using yarn run start and it puts it at http://localhost:3000, so the base url would be: "baseUrl": "http://localhost:3000/"

The api still runs on port 8000.

Reply:
Good point. We could add a note that if you use a different port for your local server, you can override settings by providing a `--baseUrl` option to the run command.

Reply:
Oh i see you already had a note in the readme.